201-key-vault-secret-create/azuredeploy.json

```diff
@@ -54,10 +54,10 @@
             }
         },
         "secrets": {
-            "type": "array",
+            "type": "secureObject",
```

Comment:
I like this but it took me a while to wrap my head around what you're doing given the naming convention...  Can we use "secretObject" for the parameter name and "secrets" for the objectName?